arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -1068,17 +1084,9 @@
           "readOnly": true
         },
         "provisioningState": {
```

Comment:
Okay we will add them. Out of curiosity, what is the rationale that some services (including ARM itself) are not modeling `provisioningState` as enums? It seems that having them as enums only gives users more context but having them as strings would give server side more flexibility and avoid breaking changes in the future.